misc/dbt-materialize/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # dbt-materialize Changelog
 
-## Unreleased - 2022-10-16
+## Unreleased - 2022-10-28
+
+* Return the correct connection keys to the user on `dbt debug`.
```

Comment:
```suggestion
* Add `cluster` to the connection parameters returned on `dbt debug`.
```

Reply:
Didn't we simply not return the `cluster` parameter? As in, there wasn't anything "wrong" with what was returned, we just added a Materialize-specific connection parameter.

```nofmt
Connection:
  host: materialized
  port: 6875
  user: materialize
  database: materialize
  schema: public
  search_path: None
  keepalives_idle: 0
  sslmode: None
  Connection test: [OK connection ok]
```

Reply:
Correct, nothing wrong, just bringing through the other parameters that we grab from dbt-postgres.

Reply:
Approved the PR, but would still adjust the changelog!

Reply:
Adjusted in release PR.